sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -122,6 +122,7 @@ object FunctionRegistry {
     expression[Tanh]("tanh"),
     expression[ToDegrees]("todegrees"),
     expression[ToRadians]("toradians"),
+    expression[Logarithm]("logarithm"),
```

Comment:
we should name this log to make it compatible. (you'd need to make the logarithm expression itself being able to handle both)